src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -3,10 +3,11 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import 'date-fns';
-import DateFnsUtils from '@date-io/date-fns';
-import { makeStyles } from '@material-ui/core';
-import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
+import makeStyles from '@mui/styles/makeStyles';
+import { TextField } from '@mui/material';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { LocalizationProvider } from '@mui/x-date-pickers//LocalizationProvider';
+import { DesktopDatePicker } from '@mui/x-date-pickers/DesktopDatePicker';
```

Comment:
These changes seem to cause warnings during the `yarn build` command:
```
(!) Unresolved dependencies
https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency
@mui/x-date-pickers/AdapterDateFns (imported by "src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js")
@mui/x-date-pickers/DesktopDatePicker (imported by "src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js")
@mui/x-date-pickers//LocalizationProvider (imported by "src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js")
```

Also, the `date-fns` dependency seems to be no longer used, could you remove it with `yarn remove date-fns` please ?